crates/bevy_asset/src/server/mod.rs

```diff
@@ -131,9 +131,9 @@ impl AssetServer {
     }
 
     /// Retrieves the [`AssetSource`] for the given `source`.
-    pub fn get_source<'a>(
+    pub fn get_source<'a, 'b>(
         &'a self,
-        source: impl Into<AssetSourceId<'a>>,
+        source: impl Into<AssetSourceId<'b>>,
     ) -> Result<&'a AssetSource, MissingAssetSourceError> {
```

Comment:
Can't this be written as (I can't do a suggestion because it touches unchanged lines)

```rust
    pub fn get_source<'a>(
        &self,
        source: impl Into<AssetSourceId<'a>>,
    ) -> Result<&AssetSource, MissingAssetSourceError> {
```